tests/5.1/scope/test_scope_private_construct.c

```diff
@@ -0,0 +1,37 @@
+//--------------- test_scope_private_construct.c -----------------------------//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// This test checks that the scope private construct clause is properly working.
+// The test itself passes a test integer into the scope pragma and ensures that
+// all changes made to it are not kept outside of the scope region.
+//----------------------------------------------------------------------------//
+
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+int test_scope(){
+	int errors = 0;
+	int test_int = 1;
+	#pragma omp task shared(test_int)
+	{
+		#pragma omp scope private(test_int)
```

Comment:
_I know that this has already merged. – Still as remark to this pull request:_

This **fails** with GCC mainline with:
```
tests/5.1/scope/test_scope_private_construct.c: In function ‘test_scope’:
tests/5.1/scope/test_scope_private_construct.c:22:25: error: ‘scope’ region may not be closely nested
inside of work-sharing, ‘loop’, explicit ‘task’, ‘taskloop’, ‘critical’, ‘ordered’, ‘master’,
or ‘masked’ region
   22 |                 #pragma omp scope private(test_int)
      |                         ^~~
```

Cf. https://www.openmp.org/spec-html/5.1/openmpse30.html (last item)
> *  A **scope** region may not be closely nested inside a worksharing, **loop**, **task**, **taskloop**, **critical**, **ordered**, **atomic**, or **masked** region.